### PR TITLE
DCAPP-27 Comment APIs

### PR DESCRIPTION
**Pull request by sam-ludlow**
https://io1dev.atlassian.net/browse/DCAPP-27

### CODE REVIEW

File: api.swagger.yaml [hunk 1]
```diff
@@ -3652,6 +3652,37 @@ paths:
...
         '404':
           description: No content with that UUID was found.
+  /comments:
+    post:
+      tags:
+        - comments
+      description: 'Provision a new comment.'
+      responses:
+        '201':
+          description: 'The promotion was created successfully.'
```
[the remark this answers is not in the document]
Comment (sam-ludlow) [reply api.swagger.yaml]: done

File: api.swagger.yaml [hunk 2]
```diff
@@ -11938,20 +11998,115 @@ components:
...
         - likes_count
+    comment-schema-create:
+      allOf:
+        - $ref: '#/components/schemas/comment-schema-common'
+        - type: object
+          properties:
+            parent_uuid:
+              type: string
+              format: uuid
+              description: 'This is the Invotra UUID of the parent of the comment. It may be a post or comment.'
```
[the remark this answers is not in the document]
Comment (sam-ludlow) [reply api.swagger.yaml]: We need to talk

File: api.swagger.yaml [hunk 3]
```diff
@@ -11938,20 +11998,115 @@ components:
...
+              example: '01234567-89ab-cdef-1234-56789abcdef0'
+            category:
```
[the remark this answers is not in the document]
Comment (sam-ludlow) [reply api.swagger.yaml]: done

File: api.swagger.yaml [hunk 4]
```diff
@@ -11938,20 +11998,115 @@ components:
...
+              type: object
+              properties:
+                uuid:
+                  type: string
+                  format: uuid
+                  description: 'This is the Invotra UUID of the comment category.'
+                  example: '01234567-89ab-cdef-1234-56789abcdef0'
+              required:
+                - uuid
+            attachments:
```
[the remark this answers is not in the document]
Comment (sam-ludlow) [reply api.swagger.yaml]: done

**Comment by MireilleHelle** [comment]
Closing as this was merged in #254 alongside other project changes.

**Pull request closed without merging**